addons/mail/static/src/components/composer/composer.xml

```diff
@@ -21,7 +21,7 @@
                 t-on-keydown="composerView.onKeydown"
                 t-ref="root"
             >
-                <t t-if="composerView.dropZoneView">
+                <t t-if="composerView.dropZoneView and composerView.hasDropZone">
```

Comment:
no need for the new field in this version, compute for dropZoneView should be adapted instead